bootloader/grub.go

```diff
@@ -690,3 +690,43 @@ func (g *grub) BootChains(runBl Bootloader, kernelPath string) ([][]BootFile, er
 
 	return chains, nil
 }
+
+// ConstructShimEfiLoadOption returns a serialized load option for the shim
+// binary. It should be called on a UefiBootloader.
+func (g *grub) EfiLoadOptionParameters(updatedAssets []string) (description string, assetPath string, optionalData []byte, err error) {
```

Comment:
Looks like the former `getGrubShimBinaryFullPath` helper was merged into this function -- great! I think the doc comment could use some more information about what the given `updatedAssets` are how they relate.

This approach seems much more robust than what I had done before -- thanks!